frontend/yarn.lock

```diff
@@ -0,0 +1,5890 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Max and I spoke about this on Slack, and I tracked down the initial case which set precedent: https://hackclub.slack.com/archives/C3P3J5N9J/p1484247963000014

This references a bug in Netlify being the cause, and because we're not using Netlify anymore we should be A OK.